src/assets/i18n/en.json

```diff
@@ -273,6 +273,12 @@
         "label": "Miles remaining to destination",
         "pattern": "should should only be numbers!"
       },
+      "vesselDraft": {
+        "name": "vesselDraft",
+        "label": "vessel Draft",
```

Comment:
Shouldn't this be in title case like `Vessel Draft` (or `Vessel draft`) if it is a label.  It seems weird to use camel case for a human readable string at least.